### DISCUSSION TITLE
Clean and standardize PipelineC repo.

### DISCUSSION THREAD

**Discussion by bartokon** [JulianKemmerer/PipelineC#284] [General]
Repo readability is really low. I suggest to move all .h files to some kind of headers folder and iee, io, axi folders to merge them. Do we really need one extra layer-directory for dvi/dvi_pmod.c? fixed/q0_23.h?

**Comment by JulianKemmerer** [comment]
Making a `headers` directory sounds fine to me
And then move dirs+files like you said `headers/axi/axis.h` for ex.
`dvi/` dir was to match `vga/` which did end up having a few files in it
`fixed/` is supposed to be filled in with various fixed point type implementations (if they dont get autogenerated otherwise :-/)

**Comment by JulianKemmerer** [comment]
I might prefer `include` as dir name over `headers`
ex. `include/axi/axis.h`

**Comment by ghost** [comment]
Can we roll this out?

**Comment by JulianKemmerer** [comment]
I think it needs some clarification still @Datavenia 

Is the goal to move directories with .h files into `include/<name>/<file>.h` ?

I am not in a rush to get to this

But if you put together a PR with the change I can try and test it after the big cleanup PR thats also pending probably

**Comment by JulianKemmerer** [comment]
The headers now live under include/.
